### PR TITLE
Remove deprecated aws code + docs references

### PR DESCRIPTION
**Pull request by colhom**
\cc @aaronlevy @robszumski

### CODE REVIEW

**Comment by aaronlevy** [comment]
Can you also drop the travis tests (looks like they were only for the aws code)

File: Documentation/configure-kubectl.md [hunk 1]
```diff
@@ -2,9 +2,8 @@
 
 `kubectl` is a command-line program for interacting with the Kubernetes API. The following steps should be done from a local workstation to configure `kubectl` to work with a new cluster.
 
-To quickly launch a cluster, follow these guides for [AWS][kube-aws], [Vagrant][vagrant-multi], or [full step-by-step][manual] instructions. 
+To quickly launch a cluster, follow these guides for [Vagrant][vagrant-multi] or [full step-by-step][manual] instructions. 
 
-[kube-aws]: kubernetes-on-aws.md
```
Comment (aaronlevy) [inline Documentation/configure-kubectl.md]: Maybe instead point to the new location rather than removing all together?

**Comment by aaronlevy** [comment]
Also, might want to leave a README.md under `multi-node/aws` pointing to the new repo / releases / docs

File: Documentation/configure-kubectl.md (hunk 1 above)
Comment (robszumski) [inline Documentation/configure-kubectl.md]: Yeah that would be a good idea

**Comment by colhom** [comment]
@robszumski @aaronlevy ptal.

**Comment by aaronlevy** [comment]
lgtm

**Pull request merged**